cast/src/lib.rs

```diff
@@ -628,6 +628,183 @@ where
         Ok(receipt)
     }
 
+    pub async fn events(&self, hash: String, chain: Chain, api_key: String) -> eyre::Result<String> {
```

Comment:
We have a type for transaction hashes: https://docs.rs/ethers/latest/ethers/types/struct.TxHash.html